src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static String color;

```

Comment:
Why color is class-level?
```suggestion
```

Reply:
To avoid creating this object each time when i call getRandomBall() method.

Reply:
You **can not** avoid creating this object each time when i call getRandomBall() method

Reply:
My comment "avoid creating this object each time" was about ColorSupplier, not color. @Hero284